src/knex-tables.d.ts

```diff
@@ -59,22 +65,41 @@ declare module "knex/types/tables" {
     breaches_resolved: null | unknown;
     // TODO: Find unknown type
     waitlists_joined: null | unknown;
-    // TODO: Find unknown type
-    breach_stats: null | unknown;
+    breach_stats: null | {
+      passwords: { count: number; numResolved: number };
+      numBreaches: {
+        count: number;
+        numResolved: number;
+        numUnresolved: number;
+      };
+      monitoredEmails: { count: number };
+    };
     // TODO: Find unknown type
     monthly_email_at: null | unknown;
     // TODO: Find unknown type
     monthly_email_optout: null | unknown;
-    // TODO: Find unknown type
-    breach_resolution: null | unknown;
+    breach_resolution:
+      | null
+      | ({
+          useBreachId: boolean;
+        } & Record<
+          SubscriberEmail.email,
+          Record<
+            BreachRow.id,
+            {
+              isResolved: boolean;
+              resolutionsChecked: Array<
+                (typeof BreachDataTypes)[keyof typeof BreachDataTypes]
```

Comment:
I wasn't aware of this previously, but apparently we do keep track for each breach specifically what data classes the user has resolved - in the `subscribers` table. So this is also the column we'll have to update when the user resolves something in the guided flow.